cmd/promtool/main.go

```diff
@@ -133,6 +133,11 @@ func main() {
 	dumpPath := tsdbDumpCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
+	importCmd := app.Command("import", "[Experimental] import samples from input and produce TSDB block. Please refer to the storage docs for more details.")
+	importDbPath := importCmd.Flag("output", "output directory for generated block").Default(".").String()
```

Comment:
```suggestion
	importDBPath := importCmd.Flag("output", "Output directory for generated block(s).").Default(".").String()
```
Reason: https://github.com/golang/go/wiki/CodeReviewComments#initialisms